tests/unit/helpers/language-helper-test.php

```diff
@@ -47,7 +36,6 @@ protected function set_up() {
 	 * @covers ::__construct
 	 */
 	public function test_constructor() {
-		$this->assertInstanceOf( Japanese_Support_Conditional::class, $this->getPropertyValue( $this, 'japanese_conditional' ) );
 	}
```

Comment:
I think it's better to remove this `test_instructor` code block as it's actually emtpy